core/graphelier-service/db/mongo.go

```diff
@@ -22,6 +22,7 @@ type Datastore interface {
 	RefreshCache() error
 	GetSingleOrderMessages(instrument string, SODTimestamp int64, EODTimestamp int64, orderID int64) ([]*models.Message, error)
 	GetTopOfBookByInterval(instrument string, startTimestamp uint64, endTimestamp uint64, maxCount int64) (results []*models.Point, err error)
+	GetMessagesWithinInterval(instrument string, startTimestamp uint64, endTimestamp uint64) (results []*models.Message, err error)
```

Comment:
From what i remember this can return more than an interval's worth of messages so it might need to be renamed. @Lercerss can you confirm?

Reply:
I actually have something very similar in #184 with a [different name](https://github.com/Lercerss/graphelier/pull/184/files#diff-1717a8460c15d66b9eab79dbc9668be6R19)